package.json

```diff
@@ -20,9 +20,9 @@
     "cypress-cucumber-preprocessor": "^4.3.0",
     "cypress-file-upload": "^4.1.1",
     "mocha": "^9.1.2",
-    "mochawesome": "^6.3.1",
+    "mochawesome": "^7.0.1",
     "mochawesome-merge": "^4.2.0",
-    "mochawesome-report-generator": "^5.2.0",
+    "mochawesome-report-generator": "^6.0.1",
```

Comment:
til neste gang: 
i frontend har vi egne PRs for bumping av versjoner. syns det er ryddig for å unngå for mye endringer som egentlig ikke er relatert til hverandre i samme PR